[2025-07-26T09:43:47.463+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-26T09:43:47.502+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_postgres_test.create_tables manual__2025-07-26T09:43:45.769663+00:00 [queued]>
[2025-07-26T09:43:47.520+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_postgres_test.create_tables manual__2025-07-26T09:43:45.769663+00:00 [queued]>
[2025-07-26T09:43:47.522+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-07-26T09:43:47.547+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): create_tables> on 2025-07-26 09:43:45.769663+00:00
[2025-07-26T09:43:47.561+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=261) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T09:43:47.561+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_postgres_test', 'create_tables', 'manual__2025-07-26T09:43:45.769663+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/my_dags.py', '--cfg-path', '/tmp/tmpuv1z_nc6']
[2025-07-26T09:43:47.569+0000] {standard_task_runner.py:72} INFO - Started process 266 to run task
[2025-07-26T09:43:47.579+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask create_tables
[2025-07-26T09:43:47.742+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_postgres_test.create_tables manual__2025-07-26T09:43:45.769663+00:00 [running]> on host 2f3d4b2b9190
[2025-07-26T09:43:47.925+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_postgres_test' AIRFLOW_CTX_TASK_ID='create_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T09:43:45.769663+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-26T09:43:45.769663+00:00'
[2025-07-26T09:43:47.927+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-26T09:43:47.978+0000] {my_dags.py:369} INFO - All tables created successfully
[2025-07-26T09:43:47.979+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-26T09:43:47.998+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-26T09:43:47.999+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_postgres_test, task_id=create_tables, run_id=manual__2025-07-26T09:43:45.769663+00:00, execution_date=20250726T094345, start_date=20250726T094347, end_date=20250726T094347
[2025-07-26T09:43:48.042+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-26T09:43:48.098+0000] {taskinstance.py:3895} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-07-26T09:43:48.101+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
